.tours/metrics-for-tabular-data.tour

```diff
@@ -0,0 +1,130 @@
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "The operand of the metric is a `Column`. A `Column` is an extractor function (initialized with the dimension's name) that when given a `pandas.DataFrame`, returns a `pandas.Series`. \n\nIt's set up this way because the column with the name we want may not exist in the `DataFrame`, so the extractor function will return a `Series` of `NaN`s if that happens.",
+      "line": 135
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "Drift `Metric` is initialized with the reference data.",
+      "line": 136
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "For drift metric on continuous data, we also need binning. `QuantileBinning` is the default.",
+      "line": 142
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "When everything as assembled, `pandas.DataFrame.pipe` will carry out the calculation on the `DataFrame`.",
+      "line": 149
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`pandas.DataFrame.pipe` will pipe results between calculations that return `DataFrames` as results.",
+      "line": 71
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "timeseries function here will handle how the data is chopped up.",
+      "line": 72
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`sampling_interval` specifies how often a data point is produced. It can be daily, hourly, etc. The smallest interval is a minute.",
+      "line": 79
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`evaluation_window` specifies how much data each timeseries data point contains. It can be one day's or one hour's worth of data. The smallest interval is one minute.\n\nThe timestamp of each timeseries data point is the timestamp at the end of the `evaluation_window`.\n\nEach `evaluation_window` is a right-open interval, i.e. the end time instant is excluded. ",
+      "line": 76
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "Once the data is chopped up, each portion is passed as `pandas.DataFrame` to the metric function. The `PSI` metric function accepts a `DataFrame` an returns a `float`.",
+      "line": 82
+    },
+    {
+      "file": "src/phoenix/metrics/metrics.py",
+      "description": "This is the formula for the PSI calculation inside the `Metric`. In general, a `Metric` is a wrapper for functions from third-party libraries. In this case the third-party function is `entropy` from the `scipy` library.",
+      "line": 207
+    },
+    {
+      "file": "src/phoenix/metrics/metrics.py",
+      "description": "We use cooperative multiple-inheritance for `Metric`.",
```

Comment:
I'm curious to hear about why we're using this strategy in this description

Reply:
mixins are pretty common in Python. Here is an [example](https://github.com/online-ml/river/blob/ce6fbc34b05c77d99a7475058379565444a759b3/river/metrics/cross_entropy.py#L10).